pomidor.el

```diff
@@ -486,6 +519,92 @@ TIME may be nil."
     (plist-put state :stopped (current-time)))
   (nconc pomidor-global-state (list (pomidor--make-state))))
 
+(defun pomidor-save-session ()
+  "Save the current session in a file."
+  (interactive)
+  (let ((time-asked-to-save (current-time)))
+    (pomidor-quit)
+    (plist-put (pomidor--current-state) :stopped time-asked-to-save)
+
+    (when (not (file-exists-p pomidor-save-session-file))
+      (with-temp-file pomidor-save-session-file
+        (insert (prin1-to-string (make-hash-table :test 'equal)))))
+    (let* ((file-table (pomidor--read-session))
+           (name (format-time-string "%Y-%m-%dT%H:%M:%S" time-asked-to-save))
+           (global-state (-map (lambda (pomodoro) (plist-put pomodoro :current-time time-asked-to-save))
```

Comment:
Either `dash`  should be added to `Package-Requires:` or builtin `seq-map`, `seq-filter` etc should be used.

Reply:
Do you have any problem adding a new dependency to the package? I might change to use the built ins if you want to. The `dash` syntax is more familiar to newcomers as me rsrsrs.

Reply:
For now I added the dash dependency, can change if necessary.